src/main/resources/application.yaml

```
@@ -1,3 +1,14 @@
     context-path: /api
+
+spring:
+  datasource:
+    url: jdbc:mysql://${DB_HOST:localhost}:${DB_PORT:3306}/${DB_DATABASE:thirty_day_challenge}
+    username: ${DB_USERNAME:root}
+    password: ${DB_PASSWORD:password}
+  jpa:
+    open-in-view: true
+    show-sql: true
+    hibernate:
+      ddl-auto: update
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)